services/mailer.js

```diff
@@ -0,0 +1,23 @@
+const mailgun = require('mailgun-js');
+const logger = require('../lib/log')
+require('dotenv').config()
+
+const mg = mailgun({apiKey: process.env.MAILGUN_API_KEY, domain: process.env.DOMAIN});
+
+const sendConfirmationEmail = () => {
```

Comment:
`receiver` should come in as an argument. This function needs 2 arguments, a token and a receiver.